content/articles/understanding-facial-recognition-using-local-binary-pattern-histogram-(lbph)-algorithm/index.md

```diff
@@ -0,0 +1,92 @@
+
+Using a circular neighborhood concept, which takes non-integer pixel points around the selected area. The number of appearances of LBP codes in the image is put together to form a histogram. The classification is then carried out through the calculation of the basic similarities of the histograms under comparison. 
+
+This histogram contains a description of an individual at three different levels: at a pixel-level, labels are combined in a small area to create a regional level, the regional histograms in combination build a general description of the person. 
+
+### Two modes of operation of face recognition 
+The face recognition algorithm generally operates in one of two modes: 
+
+1.	*Authentication of a facial image:* This mode does facial recognition by a 1x1 comparison. The comparison is done between an input image and a specific image within the database. In many cases, this is the face that requires authentication at the time of this mode of facial recognition.
+
+2.	*Face recognition:* in this mode, it is a 1xN, a comparison of the input face image with all the pictures that have been saved in the database to output the images of the user which conforms to the input face image.
+
+### The stages of face recognition using LBPH 
+For face recognition, the LBPH algorithm follows some steps. These steps can be carried out in two stages, as follows: 
+#### The learning of the algorithm
+The first step is the learning of the algorithm. It makes use of a data-set of images of the people to be included in the recognition process. Each image is given a unique ID as either a number or the name of a person so that the algorithm can use this information to identify the image and export the output. The pictures of the same person are always placed under the same ID. 
+#### Computational steps
+1.	**The application of the LBP operation:** is the first step of the computational steps. Here, an intermediate image has been created to better represent the original image through a sliding window concept, taking into account two parameters: the `neighbor` and the `radius`. New values are created in the form of binary by comparing the 8 `neighbor` values to the threshold value. For each `neighbor` value greater than the threshold value, the value is set to 1 and 0 for every `neighbor` value less than the threshold value. This forms a matrix of binary numbers excluding the threshold. A central value of the matrix is created by the conversion of the binary number to a decimal value which corresponds to the pixels of the original image. For a better representation of the characteristics of the original image.
```

Comment:
Assuming the reader doesn't know what the sliding window concept is, perhaps provide a link leading to a definition of this concept. Or define it briefly within the paragraph.